src/view/TypingGame.vue

```diff
@@ -342,6 +350,42 @@ function sentenceFeedback() {
 
   }
 }
+const curWPM = ref(wpm);
+async function saveAttempt() {
+  if (!authStore.isLoggedIn) {
+    console.log('User not logged in. Attempt not saved.');
+    return;
+  }
+  try {
+    const response = await fetch(`${API_URL}/api/attempt`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        characters_attempted: attemptedCharacters.value,
+        characters_missed: numOfWrongChars.value,
+        wpm: curWPM.value,
+        email: authStore.user.name
```

Comment:
Is the email supposed to be set to the users name?

Reply:
Good question, the variables here are correct, I'm not a fan of the variable naming used for this. I'm not sure if name was originally intended to be username or email but we should refactor this. @jasgcode did you intend to store username with `user.name`?